kong/db/schema/init.lua

```diff
@@ -94,8 +94,8 @@ local validation_errors = {
   -- schema error
   SCHEMA_NO_DEFINITION      = "expected a definition table",
   SCHEMA_NO_FIELDS          = "error in schema definition: no 'fields' table",
-  SCHEMA_MISSING_ATTRIBUTE  = "error in schema definition: missing attribute %s",
   SCHEMA_BAD_REFERENCE      = "schema refers to an invalid foreign entity: %s",
+  SCHEMA_CANNOT_VALIDATE    = "error in schema prevents from validating this field",
```

Comment:
Due diligence here: will said error be visible to the user somewhere?

Reply:
The error in case is a crash in the schema library caught by `pcall` (a840a61 has the full story). I chose not to expose the pcall error as that won't really tell the user what the problem is (and could make the user think there was a bug in Kong instead), but at least this SCHEMA_CANNOT_VALIDATE message will be matched to the specific key that caused it, so at least that's _some_ indication on where their problem is.

(The pcall error could help _us_ to explain the user what their problem is, though...)

Reply:
Maybe worth exposing it in a debug log?

Reply:
ohh, good idea

Reply:
Done!